src/components/testCaseLanding/common/ActionCenter/ActionCenter.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from "react";
+import "twin.macro";
+import "styled-components/macro";
+import { MenuItem } from "@mui/material";
+import { Select, TextField } from "@madie/madie-design-system/dist/react";
+import SearchIcon from "@mui/icons-material/Search";
+import InputAdornment from "@material-ui/core/InputAdornment";
+interface ActionCenterProps {}
+
+const filterByOptions = ["Status", "Group", "Title", "Description"];
+const searchInputProps = {
+  startAdornment: (
+    <InputAdornment position="start">
+      <SearchIcon />
+    </InputAdornment>
+  ),
+};
+
+export default function ActionCenter(props: ActionCenterProps) {
```

Comment:
thinking about the component name- I think `ActionCenter` will be global actions like edit, delete, export, import, run, etc which will be implemented in future stories. are you thinking to add them in this component

Reply:
Yes, I looked at the designs and those buttons and these search fields are in the same row. So I thought we could isolate them.

From Figma 
![image](https://github.com/user-attachments/assets/eda9a069-128e-4058-91d9-623f18b56a03)